maistra/scripts/update-deps.sh

```diff
@@ -94,15 +102,34 @@ function copy_files() {
 }
 
 function run_bazel() {
-  for config in s390x ppc x86_64; do
-    bazel --output_base="${OUTPUT_BASE}" build --nobuild --config="${config}" //src/... //test/... //extensions/...
+  # Fetch stats_plugin just to load emsdk and net_zlib dependencies
+  bazel --output_base="${OUTPUT_BASE}" fetch //extensions/stats:stats_plugin || true
+
+  # Patch emsdk and net_zlib
```

Comment:
I' m convinced I had deleted slinks yesterday but they're hard to die :-)

Yes, a  patch function can be called after fetching everything.

Reply:
Actually, patches has to be applied after loading vendor/emsdk and before fetching the big chunk, as this is necessary to avoid errors in bazel processing, even if "no_build" is specified.